scripts/flatten.py

```diff
@@ -0,0 +1,130 @@
+import argparse
+import json
+from pathlib import Path
+
+
+def recursive_search(path: Path):
```

Comment:
FYI - you could use `glob` for this:
https://stackoverflow.com/a/14798263/131896